docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-requirements.rst

```diff
@@ -125,7 +127,9 @@ to optimize how Determined interacts with PBS:
 
    -  A PBS queue with no GPUs is identified as an AUX resource pool.
 
-   -  The Determined default resource pool is set to the PBS default queue.
+   -  The Determined default resource pool is set to the PBS default queue. Override this default
+      using the :ref:`pbs section <cluster-configuration-slurm>` ``default_compute_resource_pool``
```

Comment:
Should that read '<cluster-configuration-pbs>'?

Reply:
I think your comment got truncated?   Not sure what you are asking.

Reply:
Yeah, something amiss there. 

You have cluster-configuration-slurm, but shouldn't that be cluster-configuration-_pbs_

Reply:
Nope.   This the resource_manager type is a single section in the master.yaml docs

.. _cluster-configuration-slurm:

   -  ``type: slurm`` or ``pbs``: The HPC launcher submits tasks to a Slurm/PBS cluster. For more
      information, see :ref:`using_slurm`.